examples/ts/src/query_proof_data.ts

```diff
@@ -0,0 +1,25 @@
+import {createApi} from "./api";
+
+/**
+ * Example of getting the proof for the particular leaf in the block.
+ */
+async function main() {
+    const api: any = await createApi()
+    // index of a leaf in the merkle trie for which the proof is generated
+    const dataIndex = 0;
+    // hash of the block from where to get the proof
+    const hashBlock = "0x693ae169131a736a88c672b313a5abbf97e7e2dc0d2a4c47a220874453260c10";
```

Comment:
Same comment as earlier. 

Maybe we add this to the readme so that the person running the example knows which fields to change?